data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUE
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Gabumon', TO_DATE('Nov 15th,2018', 'Mon DDth, YYYY'), 2, true, 8);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', TO_DATE('Jan 7th,2021', 'Mon DDth YYYY'), 1, false, 15.04);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', TO_DATE('May 12th,2017', 'Mon DDth YYYY'), 5, true, 11);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander', TO_DATE('Feb 8th,2020','Mon DDth YYYY'), 0, false, 11);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Plantom', TO_DATE('Nov 15th,2021','Mon DDth YYYY'), 2, true, 5.7);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Squirtle', TO_DATE('Apr 2nd,1993','Mon DDth YYYY'), 3, false, 12.13);
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Angemon', TO_DATE('Jun 12th,2005','Mon DDth YYYY'), 1, true, 45);
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235705612-d2d58f9e-5b19-4616-8b0d-2cc0dc02958d.png)

- [ ] Kindly add the correct weight for these animals because these weights should be negative 🤔 as you can see in the image above ⬆️